src/app/errors.ts

```diff
@@ -17,13 +17,13 @@ export const ERROR_REPORT: {
     },
   },
 
-  // The STANDARD
+  // The STANDARD_NO_VERBOSE
   // use by default on every not expect HTTP response
   // It will report the error on google analytics and will be show to the user **only on QA and local dev environments**
-  STANDARD: {
+  STANDARD_NO_VERBOSE: {
     display: {
-      title: $localize`:@@ngOrcid.error:Oh no! An error occurred!`,
-      message: `not verbose`,
+      title: $localize`:@@ngOrcid.error:Oh no! An error occurred`,
+      message: `Internal Orcid QA error please`,
```

Comment:
I think this error message it is incomplete

Reply:
it will be completed with the action string, like this

![image](https://user-images.githubusercontent.com/2119626/90681569-d9da1380-e220-11ea-8a7b-64801f7f68b0.png)

Also does not require translation since this STANDARD_NO_VERBOSE error is only shown on QA and DEV